### DISCUSSION TITLE
如何与其他同学交流作业成果？

### DISCUSSION THREAD

**Discussion by KaygNas** [BoomingTech/Piccolo#429] [GAMES104]
如题，[smartChair](http://smartchair.org/GAMES104) 已经无法提交作业了，而又觉得自己做的作业看上去不太对，想和其他同学对一下作业看看的话，我应该怎么做？

**Comment by 1623275623** [comment]
那应该就是在这个上面交流了吧

**Comment by KaygNas** [comment]
贴一下自己 [PA2 的作业](https://github.com/KaygNas/Piccolo/tree/games104/homework02-rendering).

被两个和单位有关的问题卡了一整天，比如：
```glsl
highp ivec2 lut_tex_size = textureSize(color_grading_lut_texture_sampler, 0);
highp vec4 color = subpassLoad(in_color).rgba;
```
因为上面的 lut_tex_size 单位是 px, 以为 colo r也是 px 后来才发现 sRGB 色彩空间的值是 [0.0, 1.0]，
同样的问题也在 `texture(color_grading_lut_texture_sampler, uv)` 遇上了（uv 的取值范围也是 [0.0, 1.0]）。🫠
